Zeeguu Reader/Pods/Zeeguu-API-iOS/ZeeguuAPI/ZeeguuAPI/Bookmark.swift

```diff
@@ -45,10 +45,13 @@
 import UIKit
 
 /// The `Bookmark` class represents a bookmark. It holds the `date`, `word`, `translation` and more about the bookmark.
-public class Bookmark {
+public class Bookmark: ZGSerialization {
```

Comment:
By the way, wouldn't it make more sense to have these comments in the ZeeguuAPI iOS repository?